### ISSUE TITLE
moving from Z490 Vision D + i9-10900K to Z590 Vision D + i9-11900K hackintosh

### ISSUE THREAD

**Issue by Nishant-Kathil** [SchmockLord/Gigabyte-Z590i-Vision-D-11900k#4]
Hi @SchmockLord ,

I am moving from Z490 Vision D + i9-10900K to Z590 Vision D + i9-11900K hackintosh. Do you see any issues or features not working yet?

Would I be able to use your Z590i configs as it is?

**Comment by RobyRew** [comment]
> Hi @SchmockLord ,
> 
> 
> 
> I am moving from Z490 Vision D + i9-10900K to Z590 Vision D + i9-11900K hackintosh. Do you see any issues or features not working yet?
> 
> 
> 
> Would I be able to use your Z590i configs as it is?
> 
> 
> 
> I have Sapphire Nitro 5700XT Graphics and not eGPU

Man, your actual PC its powerful than a i9 11900K.

You are basically trying to downgrade from 10 cores to 8 cores? For me it doesn't have any sense, sorry.

**Comment by Nishant-Kathil** [comment]
thats true RobyRew, I had an issue with Z490 Vision D + i9-10900K. the PC keeps freezing randomly. I have been told that the motherboard and CPU have an issue

So rather than going back to the same build, I am trying to get a new build going

**Comment by RobyRew** [comment]
Then, my recommendation its to buy a Z590 VISION D with the i9 10900K

**Comment by gtrooper** [comment]
> Then, my recommendation its to buy a Z590 VISION D with the i9 10900K

Good recommendation... IF it weren't for the fact that you are BADLY missing those PCIe 4.0 SSD Speeds .... For Some people that is huge ... Gigabyte Z490 with 11900K is maybe a more "money saving" decision !

**Comment by SchmockLord** [comment]
I have made the switch from Z490/10900k to Z590/11900k. I don't feel any difference. Multicore Performance is roughly the same. Single Core is higher. But I needed those two Thunderbolt ports, Z490 mini ITX boards had maximum one. And I wanted PCI4.0. So I think there are good reasons to make the switch.

**Comment by qixiang109** [comment]
is it possible to install 10.13.6 on Z590？i need to run cuda so the latest compatible system version is 10.13.6…

**Comment by MenaNegm** [comment]
I'm going to buy a new computer after Z370+9900k,
So Gigabyte Z490+10900k or Z490+11900k or Z590+10900k or Z590+11900k, what is your recommendation?

**Comment by SchmockLord** [comment]
I would wait for Alder Lake Z690 etc. 

If you need the iGPU, then Z490+10900k. Because when you are mixing Z590 with 10900k, you cannot use Thunderbolt and also the m.2 Slot under the CPU.

If you don't need the iGPU, Z590+11900k.

**Comment by arpaterson** [comment]
While your attention is here, could I pick your brain? 
I am looking to do one last hack, primarily for audio workstation purposes. I am proficient enough in hackintoshing but wanted to ask you about hardware choise.

Currently running an 8700k/asus z370i/radeonVII with no thunderbolt, and I wanted to gain TB3 or 4 for ultra low latency audio, which means a new motherboard and cascading costs anyway, so an update of everything else too.
The extra cores on 10900k attracted me for supporting many channels/plugins also. 
Detail: monitoring in-the-box with virtual guitar amps and effects chains that are replicas of my real hardware guitar rack plus virtual band and some mix/master too, all realtime. its just a creative path that works for me).

I am unsure if the increased single core performance of the 11900 would affect latency or overcompensate the extra 10 series cores for this purpose.

It looks like you have experience with 10900, 11900 and z490, z590 thunderbolt boards from asrock and gigabyte. The same shortlist I have. Do you have any recommendation? I had been leaning toward your 10900/asrock z490 build + TB firmware flash as it seems very complete and stable, but you seem to like this new z590 for more detailed reasons. Can retain RVII and tolerate no hotplug, but do need to get to a no-crash state.

ATX or ITX isn't too much concern, but in a future life the intel Xe and extra pcie slots would make for a better 'retirement' as later in life I move my old systems to linux server/projects and much less GPU.
